forem/forem
# [15-min-fix] Fix removing cover img from v2 editor and dashboard link

## What type of PR is this? (check all applicable)

## Description
This fixes two things:
1. Changes the "Dismiss" button in `/:article/delete_confirm` to navigate back to the dashboard, where the delete option is found.
2. Fixes the cover/main image removal for the v2 / default editor

## Related Tickets & Documents
N/A

## QA Instructions, Screenshots, Recordings
1. Comment out line 59 in `app/models/article.rb`: `validates :main_image, etc etc`
2. Go to /new with the v2 editor
3. Add a new cover image
4. Save the draft
5. Edit the draft
6. Remove the cover image
7. See that it removes properly

### UI accessibility concerns?
Nope!

## Added tests?
- [x] No, and this is why: I'm not sure whether I should write an E2E test or a Jest test...

## [Forem core team only] How will this change be communicated?

- [x] This change does not need to be communicated, and this is why not: just a simple bug fix!

## [optional] Are there any post deployment tasks we need to perform?
Nope

## [optional] What gif best describes this PR or how it makes you feel?

![April Fools!](https://media.giphy.com/media/3osxYk30y7aTinMNs4/giphy-downsized.gif)
